water_neuron/src/main.rs

```diff
@@ -333,6 +335,30 @@ async fn icp_to_nicp(arg: ConversionArg) -> Result<DepositSuccess, ConversionErr
     check_postcondition(water_neuron::conversion::icp_to_nicp(arg).await)
 }
 
+#[update]
+async fn cancel_withdrawal(neuron_id: NeuronId) -> Result<MergeResponse, String> {
+    reject_anonymous_call();
+
+    let neuron_ids: Vec<Option<NeuronId>> = read_state(|s| {
```

Comment:
why not a vec<NeuronId>?

Reply:
? It is a vec

Reply:
I mean a vec<NeuronId> why add optional neurons? it should always be set.